RiotSharp/TournamentRiotApi.cs

```diff
@@ -32,20 +33,23 @@ public class TournamentRiotApi : ITournamentRiotApi
         private static TournamentRiotApi instance;
 
         private readonly IRateLimitedRequester requester;
+        private string tournamentRootUrl;
 
         #endregion
 
-        private TournamentRiotApi(string apiKey, int rateLimitPer10s, int rateLimitPer10m)
+        private TournamentRiotApi(string apiKey, int rateLimitPer10s, int rateLimitPer10m, bool useStub = false)
```

Comment:
But how? The change is constructor related. I could update the related static methods and the DI options method.